src/demo/app/fancy-form/fancy-field-demo.component.ts

```diff
@@ -0,0 +1,264 @@
+            &lt;div class="fancy-hint"&gt;It doesn't have to be your absolute favorite, but you can only pick one.&lt;/div&gt;
+          &lt;/prx-fancy-field&gt;
+        </pre>
+        Example:
+        <prx-fancy-field [model]="model" name="color" label="Favorite color" [select]="colors">
+          <div class="fancy-hint">It doesn't have to be your absolute favorite, but you can only pick one.</div>
+        </prx-fancy-field>
+      </aside>
+      
+      <aside>
+        <h2><code>textarea</code></h2>
+        <p>Use textarea for longer text fields.</p>
+        Usage:
+        <pre class="code">
+          &lt;prx-fancy-field textarea [model]="model" name="script" label="Your script"&gt;
+            &lt;div class="fancy-hint"&gt;
+              This sponsor does not require final script approval.
+              Save your script here for storage and convenience.
+            &lt;/div&gt;
+          &lt;/prx-fancy-field&gt;
+        </pre>
+        Example:
+        <prx-fancy-field textarea [model]="model" name="script" label="Your script">
+          <div class="fancy-hint">
+            This sponsor does not require final script approval.
+            Save your script here for storage and convenience.
+          </div>
+        </prx-fancy-field>
+      </aside>
+      
+      <aside>
+        <h2><code>.nested</code></h2>
+        <p>Use content projection with the <code>.nested</code> selector to include anything in a fancy-field. Maybe a date picker?</p>
+        Usage:
+        <pre class="code">
+          &lt;prx-fancy-field label="Arrival"&gt;
+            &lt;div class="fancy-hint"&gt;When should we expect you?&lt;/div&gt;
+            &lt;prx-datepicker [date]="model.arrival" (onDateChange)="model.set('arrival', $event)" [changed]="model.changed('arrival')"&gt;
+            &lt;/prx-datepicker&gt;
+            &lt;prx-timepicker [date]="model.arrival" (onTimeChange)="model.set('arrival', $event)" [changed]="model.changed('arrival')"&gt;
+            &lt;/prx-timepicker&gt;
+          &lt;/prx-fancy-field&gt;
+        </pre>
+        Example:
+        <prx-fancy-field label="Arrival">
+          <div class="fancy-hint">When should we expect you?</div>
+          <prx-datepicker [date]="model.arrival" (onDateChange)="model.set('arrival', $event)" [changed]="model.changed('arrival')">
+          </prx-datepicker>
+          <prx-timepicker [date]="model.arrival" (onTimeChange)="model.set('arrival', $event)" [changed]="model.changed('arrival')">
+          </prx-timepicker>
+        </prx-fancy-field>
+      </aside>
+
+      <aside>
+        <h2>no <code>model</code></h2>
+        <p>If model is undefined or not provided, the field will be disabled.</p>
+        Usage:
+        <pre class="code">
+          &lt;prx-fancy-field textinput name="nada" label="Nope"&gt;
+            &lt;div class="fancy-hint"&gt;This field is disabled because there is no model.&lt;/div&gt;
+          &lt;/prx-fancy-field&lt;
```

Comment:
closing bracket points the wrong way 
![image](https://user-images.githubusercontent.com/16007382/27977220-b0ba1218-6337-11e7-92c7-50377316fb17.png)